sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -49,12 +49,6 @@
 
 
   <dependencies>
-    <dependency>
```

Comment:
Is there a particular reason we set the new version on every single child Spark Connector instead of updating it in the parent POM here?

Reply:
I'd say for now, let's just get this merged in and deal with any clean up later given this is blocking any PR running the Core pipeline

Reply:
No - I tried figuring out why still scala 2.12.10 was used - it was due to the incorrect <sclaaVersion> tag - not because pom.xml of children overrules parent pom.xml